specification.md

```diff
@@ -440,6 +440,35 @@ an individual job status callback or not. To remove the callback, set it
 to `null`.
 
 
+<a name="jobexecutor-list"></a>
+```java
+List<String> list()
+```
+
+Return a list of job IDs which are known to this executor instance.  The
```

Comment:
```suggestion
Return a list of native job IDs which are known to this executor instance.  The
```

Do we want to call out explicitly that it is native IDs that are returned? Have we defined what a native ID is in the spec yet?